### PR TITLE
Add all features

### PR DESCRIPTION
**Pull request by ybadr99**
**Team group**
- [yousef badr](https://github.com/ybadr99)
- [Festus Enning](https://github.com/Enning94)
- [kelvin Irungu](https://github.com/Kel-nana)

**Project documentation**
https://drive.google.com/file/d/1mMpsLtebb6hJEZQBeJR5Ha_Cy1uC42mj/view?usp=sharing
**Features**
- items
- books
- music albums
- games
- genres
- labels
- authors
- **Databases**
- **unit test**

### CODE REVIEW

File: spec/author_spec.rb
Comment (fobadara) [inline spec/author_spec.rb:1]: [REQUIRED] Kindly include test for `add_item` as it is a requirement.

File: spec/game_spec.rb
Comment (fobadara) [inline spec/game_spec.rb:1]: [REQUIRED] Kindly include a test for `last_played_old_enoug` as it is a requirement. And ensure all tests are passing

File: spec/label_spec.rb
Comment (fobadara) [inline spec/label_spec.rb:1]: [REQUIRED] Kindly include a test for `add_item`and`to_HasH` as it is a requirement. And ensure all tests are passing.

File: classes/item.rb
Comment (fobadara) [inline classes/item.rb:1]: [REQUIRED] Kindly include tests for item class. its a requirement for this module

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✔️ Clean code
✔️ Good file structure

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @ybadr99, @Kel-nana and @Enning94  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**